openshift/templates/cachito-worker.yaml

```diff
@@ -0,0 +1,140 @@
+kind: Template
+apiVersion: v1
+metadata:
+  name: cachito-worker
+  description: >
+    cachito-worker
+
+labels:
+  build: cachito-worker
```

Comment:
nitpick: `app` instead of `build` (otherwise it shows up under `Other resources` in OpenShift console)